<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditnomvoiture">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(184, 247, 213);

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#000000;&quot;&gt;Nom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>610</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(88, 255, 158);
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 14px;
	color: black;
	/*background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
}


QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(255, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Facture</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Downloads/1564334.png</normaloff>../../../Downloads/1564334.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(88, 255, 158);
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 14px;
	color: black;
	/*background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
}


QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(255, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Statistique</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Downloads/1170718.png</normaloff>../../../Downloads/1170718.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditmatriculevoiture">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 247, 213);

</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(88, 255, 158);
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 14px;
	color: black;
	/*background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
}


QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(255, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Downloads/64579.png</normaloff>../../../Downloads/64579.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditprenomvoiture">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 247, 213);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#000000;&quot;&gt;Prenom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>651</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 247, 213);

</string>
    </property>
    <property name="locale">
     <locale language="French" country="Tunisia"/>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>130</number>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#000000;&quot;&gt;Matricule&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(88, 255, 158);
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 14px;
	color: black;
	/*background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
}


QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(255, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Downloads/45706.png</normaloff>../../../Downloads/45706.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#000000;&quot;&gt;Cin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>150</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(183, 232, 201);
</string>
    </property>
    <property name="placeholderText">
     <string>rechercher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(88, 255, 158);
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 14px;
	color: black;
	/*background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
}


QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(255, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Downloads/121113.png</normaloff>../../../Downloads/121113.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(88, 255, 158);
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 14px;
	color: black;
	/*background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
}


QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(255, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Pdf</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Downloads/PDF_file_icon.svg.png</normaloff>../../../Downloads/PDF_file_icon.svg.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditcin">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(184, 247, 213);


</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(88, 255, 158);
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 14px;
	color: black;
	/*background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
}


QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(255, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Tri</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Downloads/271743198_464449928646161_2550356874147298555_n.png</normaloff>../../../Downloads/271743198_464449928646161_2550356874147298555_n.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_suppri">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 247, 213);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;IDBATTERIE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIDBATTERIE">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 247, 213);
</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>380</y>
      <width>511</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QLineEdit" name="lineEditfacture">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 247, 213);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Le nombre de KWH consome&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>480</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 247, 213);
border-radius: 10px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;DT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;prix total&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:20px;
fpnt-weight:800;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../2A8/projet c++/logo33.jpg</normaloff>../2A8/projet c++/logo33.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>130</width>
      <height>130</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditarduino">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>640</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 247, 213);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>680</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 247, 213);
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>680</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 247, 213);
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>640</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(53, 191, 129);

	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 14px;
	color: black;
	/*background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
}


QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(255, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Confirmer</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Downloads/confirm-16.png</normaloff>../../../Downloads/confirm-16.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>956</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
